TidyChat/TidyChatPlugin.cs

```diff
@@ -256,7 +256,9 @@ private void OnChat(XivChatType type, int timestamp, ref SeString sender, ref Se
         }
 
         var showEverythingElse = false;
-        if (chatType is ChatType.System && Configuration.ShowEverythingElse)
+        if ((chatType is ChatType.System && Configuration.ShowEverythingElse) ||
+            (chatType is ChatType.Crafting && Configuration.ShowAllOtherCrafting) ||
+            ((chatType is ChatType.Gathering || chatType is ChatType.GatheringSystem) && Configuration.ShowAllOtherGathering))
```

Comment:
oh hey I just learned that this is valid C#

```suggestion
            (chatType is ChatType.Gathering or ChatType.GatheringSystem && Configuration.ShowAllOtherGathering))
```